.github/workflows/publish-ghcr-container.yaml

```diff
@@ -0,0 +1,53 @@
+name: Publish multiarch images on ghcr.io
+on:
+  push:
+    tags:
+      - '*'
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  docker:
+    name: Build and push postgres-operator multiarch images
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: image name
+        id: image
+        run: |
+          VERSION=`git describe --tags --always --dirty`
+          TAG=${VERSION}
+          IMAGE=${{ env.REGISTRY }}/acid/postgres-operator:${TAG}
+          echo "NAME=$IMAGE" >> $GITHUB_OUTPUT
+          base_arm_image=alpine:3.15
+          sed -i '' 's/{base_image}/${base_arm_image}/g' ./docker/Dockerfile-multi-arch
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to GHCR
+        uses: docker/login-action@v2
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Run unit tests
+        run: make test
+
+      - name: Build and push
+        uses: docker/build-push-action@v3
+        with:
+          push: true
+          platforms: |
+            linux/amd64
+            linux/arm64
+          file: ./docker/Dockerfile-multi-arch
+          tags: "${{ steps.image.outputs.NAME }}"
```

Comment:
If you add `build-args: [base-image=alpine:3.15]`. You can control ARG base-image in Dockerfile

Reply:
Thanks for the tip.